config/default/configmap/config.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kfserving-config
+data:
+  TensorflowServingImage: tensorflow/serving
+  SklearnServingImage: gcr.io/kfserving/sklearnserver
+  XgboostServingImage: gcr.io/kfserving/xgbserver
```

Comment:
We need version also? Also should it be part of model deployment yaml (if users want to use a specific version of TF etc? If not specified, we choose default.  I can have one model which needs TF version X, and another one which needs TF version Y?

Reply:
I think our model spec already supports this, that is `RuntimeVersion` field. Are you suggesting we allow user to configure default version here?